### DISCUSSION TITLE
Question: Refferencing issue keys in changelog from external issue tracker

### DISCUSSION THREAD

**Discussion by keinsell** [semantic-release/semantic-release#2344] [Configuration]
Hi, I'm using `semantic-release` in my project and I wish to add backlinks to my issue keys ex. `LIT-1337` which are related to Linear/Jira or any other issue tracker. I have seen somewhere documentation for that but I was searching for like one hour to find it again and I did not succeed, can somebody drop little resources for doing such a thing?

Thanks everyone ❤️

**Comment by travi** [comment]
you are most likely looking to configure the `@semantic-release/release-notes-generator` plugin: https://github.com/semantic-release/release-notes-generator#options

**Reply by keinsell** [reply]
So I need to create my own [`conventional-changelog`](https://github.com/conventional-changelog/conventional-changelog) preset, I'm right?
